recipes/android-ndk/all/conanfile.py

```diff
@@ -336,6 +336,7 @@ def package_info(self):
         self.env_info.CMAKE_FIND_ROOT_PATH_MODE_LIBRARY = "BOTH"
         self.env_info.CMAKE_FIND_ROOT_PATH_MODE_INCLUDE = "BOTH"
         self.env_info.CMAKE_FIND_ROOT_PATH_MODE_PACKAGE = "BOTH"
+        self.cpp_info.includedirs = []
```

Comment:
I think this does not work because there are some return statements earlier. You should not have to `mkdir include` if you move this line before `if not hasattr(self, "settings_target"):`

Reply:
Done. Thank you!